client/my-sites/plugins/plugin-install-button/index.jsx

```diff
@@ -1,116 +1,142 @@
-			analytics.tracks.recordEvent( 'calypso_plugin_install_click_from_plugin_info', {
-				site: this.props.selectedSite,
-				plugin: this.props.plugin.slug
+			recordGAEvent( 'Plugins', 'Install on selected Site', 'Plugin Name', plugin.slug );
+			recordEvent( 'calypso_plugin_install_click_from_plugin_info', {
+				site: selectedSite,
+				plugin: plugin.slug
 			} );
 		}
-	},
+	}
 
-	updateJetpackAction: function() {
-		analytics.ga.recordEvent( 'Plugins', 'Update jetpack', 'Plugin Name', this.props.plugin.slug );
-		analytics.tracks.recordEvent( 'calypso_plugin_update_jetpack', {
-			site: this.props.selectedSite,
-			plugin: this.props.plugin.slug
+	updateJetpackAction = () => {
+		const {
+			plugin,
+			selectedSite,
+			recordGoogleEvent: recordGAEvent,
+			recordTracksEvent: recordEvent } = this.props;
+		recordGAEvent( 'Plugins', 'Update jetpack', 'Plugin Name', plugin.slug );
+		recordEvent( 'calypso_plugin_update_jetpack', {
+			site: selectedSite,
+			plugin: plugin.slug
 		} );
-	},
+	}
+
+	clickSupportLink = () => {
+		this.props.recordGoogleEvent( 'Plugins', 'Clicked How do I fix disabled plugin installs unresponsive site.' );
+	}
+
+	clickSiteManagmentLink = () => {
+		this.props.recordGoogleEvent( 'Plugins', 'Clicked How do I fix disabled plugin installs' );
+	}
+
+	togglePopover = ( event ) => {
+		this.refs.infoPopover._onClick( event );
+	}
 
-	getDisabledInfo: function() {
-		if ( ! this.props.selectedSite ) { // we don't have enough info
+	getDisabledInfo() {
+		const { translate, selectedSite } = this.props;
+		if ( ! selectedSite ) { // we don't have enough info
 			return null;
 		}
 
-		if ( this.props.selectedSite.options.is_multi_network ) {
-			return this.translate( '%(site)s is part of a multi-network installation, which is not currently supported.', {
-				args: { site: this.props.selectedSite.title }
+		if ( selectedSite.options.is_multi_network ) {
+			return translate( '%(site)s is part of a multi-network installation, which is not currently supported.', {
+				args: { site: selectedSite.title }
 			} );
 		}
 
-		if ( ! utils.isMainNetworkSite( this.props.selectedSite ) ) {
-			return this.translate( 'Only the main site on a multi-site installation can install plugins.', {
-				args: { site: this.props.selectedSite.title }
+		if ( ! utils.isMainNetworkSite( selectedSite ) ) {
+			return translate( 'Only the main site on a multi-site installation can install plugins.', {
+				args: { site: selectedSite.title }
 			} );
 		}
 
-		if ( ! this.props.selectedSite.canUpdateFiles && this.props.selectedSite.options.file_mod_disabled ) {
-			let reasons = utils.getSiteFileModDisableReason( this.props.selectedSite, 'modifyFiles' );
+		if ( ! selectedSite.canUpdateFiles && selectedSite.options.file_mod_disabled ) {
+			const reasons = utils.getSiteFileModDisableReason( selectedSite, 'modifyFiles' );
 			let html = [];
```

Comment:
Arrays are essentially objects, and objects can be modified without reassigning or redefining the varaible; so this should be `const html = [];` instead.